README.md

```diff
@@ -114,15 +129,22 @@ variables to be set:
 These can be gotten from the mongosh-test cluster in the Compass Atlas
 clusters.
 
-## Starting the CLI
+### Starting the CLI
 
-Via NPM:
+Via npm:
 
 ```shell
-npm start
+npm run start
 ```
 
-## Compiling
+Alternatively you can also run start inside the `cli-repl` package, if you're
+sure everything else is compiled:
+
+```shell
+cd packages/cli-repl && npm run start
+```
+
+### Compiling
 
 Compile all Typescript:
 
```

Comment:
That won't compile compass/browser, if you wanted you could include the `compile-all` which will compile typescript/browser/compass

Reply:
i forgot we changed that, thanks!